sections/kubecontext.zsh

```diff
@@ -42,14 +42,16 @@ spaceship_kubecontext() {
   local 'section_color' 'i' 'color' 'pattern'
   local len=${#SPACESHIP_KUBECONTEXT_COLOR_GROUPS[@]}
 
-  for ((i = 1; i <= $len; i+=2)); do
-    color="${SPACESHIP_KUBECONTEXT_COLOR_GROUPS[$i]}"
-    pattern="${SPACESHIP_KUBECONTEXT_COLOR_GROUPS[$i+1]}"
-    if [[ "$kube_context" =~ "$pattern" ]]; then
-      section_color=$color
-      break
-    fi
-  done
+  if [[ $len -gt 1 ]]; then
```

Comment:
do you need the `if` now or can be dropped? I imagine `$len` was `1` because it was `( () )`, but now it's just `()` by default.

Reply:
It still protects against invalid definition of the list, if someone defines it as a string by mistake, or something.

Moreover, I'd even make sure it iterates over valid pairs and ignores last unmatched value, if found.

Reply:
Let's do that instead ^ then in case of empty string or `( () )` you won't need this `if`, because it will just be an unmatched value that is ignored

Reply:
Done